app/src/main/java/org/mozilla/focus/menu/browser/BlockingItemViewHolder.kt

```diff
@@ -75,7 +77,7 @@ internal class BlockingItemViewHolder(
 
             buttonView.context.components.apply {
                 if (isChecked) {
-                    trackingProtectionUseCases.removeException(fragment.tab.id)
+                    fragment.tab.removeFromExceptions(trackingProtectionUseCases)
```

Comment:
How's the behavior different from what the use case does? And if that is needed shouldn't the use case do that? CC: @Amejia481

Reply:
Instead of passing a tab id and identifying the exception to be removed, we identify the exception from the list we fetch.

Reply:
Yeah, completely agree, it looks like this is an AC/GV issue as we should be able to remove the exception by using the `taId`, I'm trying to pinpoint the issue.

Reply:
I filed a [ticket for the GV](https://bugzilla.mozilla.org/show_bug.cgi?id=1723280)  team, in the meantime I put temporary fix on [AC](https://github.com/mozilla-mobile/android-components/pull/10728).